tracks/julia/exercises/raindrops/mentoring.md

```diff
@@ -2,32 +2,139 @@
 
 - Unrolled solutions are fine, don't require that students use loops or reduction or whatever.
 - Prefer a tuple or vector of pairs to a Dict. There's no need to add values to the Dict or lookup items by key, so a simpler container is better.
+- If performance matters, strings are slow! Profiling shows that `string(number)` and string concatenation are the biggest time sinks. Branching / lookup and remainder calculation are blazingly fast by comparison.
 
 
-## Example solutions
+## Fast lookup / tree solutions
 
-````
-[ScottPJones' solution](https://exercism.io/tracks/julia/exercises/raindrops/solutions/05647a0a81ab43fdacf8f872d113df51).
+Example solutions, roughly in order of performance. See [Luapulu's Solution](https://exercism.io/tracks/julia/exercises/raindrops/solutions/00c5eeb0e6b84b0c98a1e31ccaaca821) for benchmark results.
 
-This is about as fast as you can get without writing a tree of if/elses or a lookup table.
+### Fastest
```

Comment:
Also, it's probably better to use `*bold*` for headers and `_italic_` for subheaders because headers themselves don't get nice display in the mentor comment boxes.